scaladoc/src/dotty/tools/scaladoc/site/common.scala

```diff
@@ -58,7 +58,7 @@ final val LineSeparator = "\n"
 
 def yamlParser(using ctx: StaticSiteContext): Parser = Parser.builder(defaultMarkdownOptions).build()
 
-def loadTemplateFile(file: File)(using ctx: StaticSiteContext): TemplateFile = {
+def loadTemplateFile(file: File, titleOverride: Option[TemplateName] = None)(using ctx: StaticSiteContext): TemplateFile = {
```

Comment:
Not very important but I don’t think `titleOverride` is a good name because it does not override a previously set title, instead it provides a default title in case it is unset. I suggest using `defaultTitle` instead.